packages/fxa-profile-server/lib/img-workers.js

```diff
@@ -15,22 +15,32 @@ const WORKER_URL = config.get('worker.url');
 const ACCEPT_ENCODING_HEADER = 'identity';
 
 exports.upload = function upload(id, payload, headers) {
-  return new P(function (resolve, reject) {
+  return new P(function(resolve, reject) {
     var url = WORKER_URL + '/a/' + hex(id);
 
+    const header_excludes = config.get('worker.headers_exclude');
+    const worker_headers = Object.assign({}, headers);
```

Comment:
The newfangled way of copying is `{...headers}` 😄

Reply:
...still learning here... :D

Reply:
done.